### PR TITLE
Lepnikov ClassWork 15.03.2021 Task

### PR DESCRIPTION
**Pull request by norsulfazol**

### CODE REVIEW

File: Tasks/Lepnikov_Tasks/Task2/classwork_0315.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import math
+from time import sleep
+
+
+def print_warning(warning_str, delay=1.5):
+    """ Вывод предупреждения с задержкой на указанное время (сек.) и с возвратом каретки в начало строки.
+    """
+    print(f'\r{warning_str}', end='')
+    sleep(delay)
+    print('\r', end='')
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Lepnikov_Tasks/Task2/classwork_0315.py:13]: image

this looks very strange

File: Tasks/Lepnikov_Tasks/Task2/classwork_0315.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                if coeff_dict[i]:
+                    break
+                print_warning('Необходимо указать ненулевое число!')
+            except ValueError:
+                print_warning('Необходимо указать ненулевое целое число!')
+    title = 'Результирующая формула: %s*x**2%s%s*x%s%s=0' % (coeff_dict['a'],
+                                                             '+' if coeff_dict['b'] < 0 else '-',
+                                                             coeff_dict['b'] * (-1 if coeff_dict['b'] < 0 else 1),
+                                                             '+' if coeff_dict['c'] < 0 else '-',
+                                                             coeff_dict['c'] * (-1 if coeff_dict['c'] < 0 else 1))
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Lepnikov_Tasks/Task2/classwork_0315.py:37]: old-fashioned formatting

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good job, your score is 8.5, the solution is great and implementing of an additional validation is a very mature approach, but it looks not very good in the end.

File: Tasks/Lepnikov_Tasks/Task2/classwork_0315.py (hunk 1 above)
Comment (norsulfazol) [reply Tasks/Lepnikov_Tasks/Task2/classwork_0315.py:13]: Yes. Really. The code is executed differently in different terminals. I fixed the error and re-created the pull request.

Comment (norsulfazol) [reply Tasks/Lepnikov_Tasks/Task2/classwork_0315.py:13]: I hope that now the code will work the same in all terminals.

**Pull request merged**